crates/cli/src/main.rs

```diff
@@ -192,6 +194,13 @@ async fn main() {
             ),
             Err(err) => println!("Error during key file creation:{err}"),
         },
+        ConfCommands::PrintPublicKey => match gevulot_cli::keyfile::read_key_file(&args.keyfile) {
```

Comment:
`read_key_file()` returns `SecretKey`.

Reply:
Oh yeah, you're right, misread the code !
Should be fixed.